src/models/parallelCoordinates.js

```diff
@@ -158,13 +170,14 @@ nv.models.parallelCoordinates = function() {
         width:      {get: function(){return width;}, set: function(_){width=_;}},
         height:     {get: function(){return height;}, set: function(_){height=_;}},
         dimensions: {get: function(){return dimensions;}, set: function(_){dimensions=_;}},
+        lineTension: {get: function(){return lineTension;}, set: function(_){lineTension=_;}},
 
         // options that require extra logic in the setter
         margin: {get: function(){return margin;}, set: function(_){
-            margin.top    = typeof _.top    != 'undefined' ? _.top    : margin.top;
-            margin.right  = typeof _.right  != 'undefined' ? _.right  : margin.right;
-            margin.bottom = typeof _.bottom != 'undefined' ? _.bottom : margin.bottom;
-            margin.left   = typeof _.left   != 'undefined' ? _.left   : margin.left;
+            margin.top    =  _.top    != undefined ? _.top    : margin.top;
+            margin.right  =  _.right  != undefined ? _.right  : margin.right;
+            margin.bottom =  _.bottom != undefined ? _.bottom : margin.bottom;
+            margin.left   =  _.left   != undefined ? _.left   : margin.left;
```

Comment:
You need to do !== instead of != to ensure it's actually undefined, otherwise you can't ever set it to 0.